gsw_check_functions.c

```diff
@@ -98,9 +98,9 @@ int
 main(int argc, char **argv)
 {
         int     count = cast_m*cast_n, i, j, k, l, n;
```

Comment:
You can remove the "count = cast_m*cast_n" part; count isn't being used any more.

Reply:
"count" is still used inside the definitions of test_func, test_sub1, test_sub2 .... and during the testing it changes the value (for example at line 340 and 397). Removing the count variable can make the test functions more complex.

Reply:
OK, thanks, I missed that.